src/App.css

```diff
@@ -0,0 +1,5 @@
+body {
+  margin: 0;
+  height: 100vh;
```

Comment:
I would be safer to add `min-height`, if content do not fit it will be cut.

Reply:
Changed as suggested.